lms/djangoapps/instructor/enrollment.py

```diff
@@ -290,12 +293,13 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
                 'event_transaction_type': unicode(grade_update_root_type),
             }
         )
-        _fire_score_changed_for_block(
-            course_id,
-            student,
-            block,
-            module_state_key,
-        )
+        if not submission_cleared:
```

Comment:
This prevents the following incorrect task (in particular, `score_db_table:csm`) from being queued:

```
2017-03-29 01:32:21,897 INFO 2851 [lms.djangoapps.grades.signals.handlers] handlers.py:231 - Grades: Request async calculation of subsection grades with args: course_id:course-v1:EricX+ora_test+right_now, modified:2017-03-29 01:32:21.882929+00:00, only_if_higher:False, score_db_table:csm, score_deleted:True, signal:<django.dispatch.dispatcher.Signal object at 0x7f32faff5e90>, usage_id:block-v1:EricX+ora_test+right_now+type@openassessment+block@7e208815f1b645ed998d13ff896f5451, user_id:5, weighted_earned:0, weighted_possible:8. Task [7c6a68f5-0510-43f3-b4c4-0d914ae29832]
```